gulpfile.js

```diff
@@ -34,19 +34,20 @@ var moduleMap = {};
 for (var key in fbjsModuleMap) {
   moduleMap[key] = fbjsModuleMap[key];
 }
-var whiteListNames = [
-  'deepDiffer',
-  'deepFreezeAndThrowOnMutationInDev',
-  'flattenStyle',
-  'InitializeJavaScriptAppEngine',
-  'RCTEventEmitter',
-  'TextInputState',
-  'UIManager',
-  'View',
-];
+var whiteList = {
+  deepDiffer: 'react-native/lib/deepDiffer',
+  deepFreezeAndThrowOnMutationInDev: 'react-native/lib/deepFreezeAndThrowOnMutationInDev',
+  flattenStyle: 'react-native/lib/flattenStyle',
+  InitializeJavaScriptAppEngine: 'react-native/lib/InitializeJavaScriptAppEngine',
+  RCTEventEmitter: 'react-native/lib/RCTEventEmitter',
+  TextInputState: 'react-native/lib/TextInputState',
+  UIManager: 'react-native/lib/UIManager',
+  UIManagerStatTracker: 'react-native/lib/UIManagerStatTracker',
+  View: 'react-native/lib/View',
+};
 
-whiteListNames.forEach(function(name) {
-  moduleMap[name] = name;
+Object.keys(whiteList).forEach(function(name) {
+  moduleMap[name] = whiteList[name];
 });
 
 moduleMap['object-assign'] = 'object-assign';
```

Comment:
Can we just simplify this whole thing since you're changing to an object anyway?

```js
var moduleMap = Object.assign(
  {},
  require('fbjs/module-map'),
  {
    deepDiffer: 'react-native/lib/deepDiffer',
    ...
    'object-assign': 'object-assign',
  }
)
```

Otherwise, this seems fine. If you say the module resolution works then I believe you :) I'm a little concerned about the framework & peerDep and that screwing up the resolution, but we can tackle that later. I think the only clear solution to that is shipping the renderer in its own package, which we'll get to.